### DISCUSSION TITLE
图片打不开

### DISCUSSION THREAD

**Discussion by 303922010** [messense/aliyundrive-webdav#725] [Q&A]
V1.11.0项目我在openwrt的服务安装了一个，也在docker安装了一个，电脑用RaiDriver映射到电脑上存取文件，发现图片类的打开和选中都会卡住，其他文件打开都正常。同样的项目在群晖的docker上使用就完全正常，图片也不受影响。不知道各位有没有遇到同样情况

**Comment by leotalent** [comment]
我的也是在openwrt的docker安装的，出现同样错误，不知道怎么解决才好。

**Comment by MznJeeHK** [comment]
我的是在群晖docker安装v1.11.0版，也是视频能正常加载，图片无法加载，下载下来的图片也是格式损坏的

**Reply by 303922010** [reply]
奇怪奇怪，我群晖倒是完全正常的，就是在软路由这边出毛病，openwrt软件商店安装和docker都出现了同样问题。

**Comment by ArcCal** [comment]
用docker出现问题都不知道怎么排查，我不用docker，没你们说的奇奇怪怪的问题

**Reply by 303922010** [reply]
那用啥？我也试试。
